Automattic/jetpack
# Update/stats upgrade nudges styling

This PR implements the design of the Upgrade Nugdes that will be displayed in the Stats page. It also adds the description of each item and the "Learn more" links.

#### Changes proposed in this Pull Request:

* Style the Stats Upgrade nudges

#### Jetpack product discussion


p1HpG7-cfa-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Activate and Connect Jetpack
* Edit your wp-config.php and add define( 'JETPACK_DEV_TEST_STATS_UPGRADE_NUDGES', true );
* Visit Jetpack > Site Stats
* Check the design against p6TEKc-56L-p2
* Check the icons
* Check the texts (check for typos)
* Check the links destinations
